### ISSUE TITLE
Automatically add links to one-act plays at performing-arts.eu

### ISSUE THREAD

**Issue by lehkost** [dracor-org/einakter#51]
This would connect our metadata on one-act plays with the biggest data collection of performances. Some works at performing-arts.eu use GND "Werk" identifiers to link materials to it, for example on Kleist's "Der zerbrochne Krug":

* https://www.performing-arts.eu/work/gnd_4099275-5

Corrresponding entry in our one-act-play database:

* https://einakter.dracor.org/kleist-der-zerbrochene-krug

They use GND identifiers, which we would get from the Wikidata entries (if available). Yet they also link to Wikidata entries themselves, so maybe there's even a shortcut to connect our database to theirs.

In this case we obviously shouldn't add IDs manually, so we would need a script that regularly adds links to performing-arts.eu automatically.

In the frontend, links would appear under "Links", maybe like this:

* performing-arts.eu: [4099275-5](https://www.performing-arts.eu/work/gnd_4099275-5)

**Comment by cmil** [comment]
How would we know for which pieces performing-arts.eu has a page without probing them? This could be a bit abusive to do over and over again for pieces they do not yet or will never have a page for.

**Comment by lehkost** [comment]
Good point! And I just found out they actually **do** have a BEACON file for works, see at the bottom of [this page](https://www.performing-arts.eu/Content/dokumentation). They use ```http://d-nb.info/gnd/``` as ```#PREFIX:```, and – as next step – the Turtle representation of GND records does feature Wikidata if available. So, the above play would be referenced via:

* http://d-nb.info/gnd/4099275-5
* http://d-nb.info/gnd/4099275-5/about/lds (TTL format incl. ```owl:sameAs <http://www.wikidata.org/entity/Q486602>```)

So we actually *could* cycle through their BEACON file and automatically find entries we also have in our database.

And speaking of which, shouldn't we also provide a BEACON file on the long run? (for works and authors?)
